sessions/js-array-methods/for-each/js/index.js

```diff
@@ -0,0 +1,36 @@
+const colors = [
+	'#343434',
+	'#7FB5B5',
+	'#633A34',
+	'#354D73',
+	'#F3A505',
+	'#2A6478',
+	'#CC0605',
+	'#497E76',
+	'#D36E70',
+	'#FF2301',
+	'#4C514A',
+	'#2E3A23',
+	'#5D9B9B',
+	'#474A51',
+	'#EFA94A',
+	'#4D5645',
+	'#606E8C',
+	'#A5A5A5',
+];
+
+/*
+1: Create colored boxes from the array of hex codes using the `forEach` array method.
+  1.1: For each hex code a new `div` with the class `color-box` should be created
+  1.2: The `background-color` of the `div` should be the current hex code
+  1.3: The element should be added to the page
+*/
+
+colors.forEach();
+
+/*
+Instead of writing the callback function directly inside the forEach method, we can use a function reference.
+  1.5: Create a new function called renderColorBox
+  1.6: Copy your code from the arrow function into this new function.
+  1.7: Use the forEach method with renderColorBox to create the color boxes a second time!
+*/
```

Comment:
```suggestion
/*
2: Instead of writing the callback function directly inside the forEach method, we can use a function reference.
  2.1: Create a new function called renderColorBox
  2.3: Copy your code from the arrow function into this new function.
  2.4: Use the forEach method with renderColorBox to create the color boxes a second time!
*/
```